src-opam/dockerfile_distro.ml

```diff
@@ -18,6 +18,12 @@
 (** Distro selection for various OPAM combinations *)
 open Astring
 
+type win10_update =
+  [ `V1507 | `V1511 | `V1607 | `V1703 | `V1709 | `V1809 | `V1903 | `V1909 | `V2004 | `V20H2 | `V21H1 ] [@@deriving sexp]
+let win10_updates =
```

Comment:
Could pedantically call this `win10_releases` to cover the fact that 1507 is the original?